test/test_utils.py

```diff
@@ -0,0 +1,11 @@
+from opimodel import utils
+import pytest
+
+
+@pytest.mark.parametrize('input,output',
+                         [('hello', 'HELLO'),
+                          ('Hello:', 'HELLO_'),
+                          ('Hello: Goodbye', 'HELLO_GOODBYE'),
+                          ('Dummy 1', 'DUMMY_1')])
+def test_mangle_name_handles_various_cases(input, output):
```

Comment:
You need an extra test case for consecutive underscores as this is explicitly referenced in the mangle_name header